engine/src/main/java/org/destinationsol/world/generators/LargeSolarSystemGenerator.java

```diff
@@ -26,7 +26,7 @@
     @Override
     public SolarSystem build() {
         getSolarSystemConfigManager().loadDefaultSolarSystemConfigs();
-        setSolarSystemConfig(getSolarSystemConfigManager().getRandomSolarSystemConfig(getSolarSystemNumber() < 1));
+        setSolarSystemConfig(getSolarSystemConfigManager().getRandomSolarSystemConfig(getSolarSystemNumber() > 0));
```

Comment:
Eh?

Reply:
@Cervator this was the cause of the bug causing the first system to be harder than it should have been. I'll inline an explanation